scripts/sct_analyze_lesion.py

```diff
@@ -27,53 +27,65 @@
+
+    parser = argparse.ArgumentParser(
+        description='Compute statistics on lesions of the input binary file (1 for lesion, 0 for background). The function assigns an ID value to each lesion (1, 2, 3, etc.) and outputs morphometric measures for each lesion:'
+                    '\n- volume [mm^3]'
+                    '\n- length [mm]: length along the Superior-Inferior axis'
+                    '\n- max_equivalent_diameter [mm]: maximum diameter of the lesion, when '
+                    '\n  approximating the lesion as a circle in the axial cross-sectional plane '
+                    '\n  orthogonal to the spinal cord'
+                    '\n\nIf an image (e.g. T2w or T1w image, texture image) is provided, it computes the mean and standard deviation values of this image within each lesion.'
+                    '\n\nIf a registered template is provided, it computes:'
+                    '\n- the distribution of each lesion depending on each vertebral level and on each'
+                    '\n  region of the template (e.g. GM, WM, WM tracts).'
+                    '\n- the proportion of ROI (e.g. vertebral level, GM, WM) occupied by lesion.'
+                    '\n\nN.B. If the proportion of lesion in each region (e.g. WM and GM) does not sum '
+                    '\nup to 100%, it means that the registered template does not fully cover the '
+                    '\nlesion, in that case you might want to check the registration results.',
+        add_help=None,
+        formatter_class =argparse.RawDescriptionHelpFormatter,
+        prog=os.path.basename(__file__).strip(".py")
+    )
+    mandatoryArguments = parser.add_argument_group("\nMandatory arguments")
+    mandatoryArguments.add_argument("-m",
+                        help="Lesion mask to analyze (e.g. 't2_lesion.nii.gz')",
+                        required=True
+                        )
+    optional = parser.add_argument_group("\nOptional arguments")
+    optional.add_argument("-h",
+                          "--help",
+                          action="help",
+                          help="show this help message and exit"
+                          )
+    optional.add_argument("-s",
+                        help="Spinal cord centerline or segmentation file, which will be used to correct morphometric measures with cord angle with respect to slice. (e.g.'t2_seg.nii.gz')",
+                        required=False
+                        )
+    optional.add_argument("-i",
+                        help="Image from which to extract average values within lesions (e.g. 't2.nii.gz')",
+                        required=False
+                        )
+    optional.add_argument("-f",
+                        help="Path to folder containing the atlas/template registered to the anatomical image.",
+                        required=False
+                        )
+    optional.add_argument("-ofolder",
+                        help="Output folder (e.g. './')",
```

Comment:
better to display " instead of ' if you want to quote something. Which means that you should switch " for ' when defining the string (I'll take care of this file)